search/indexing_api.py

```diff
@@ -125,6 +129,68 @@ def _delete_item(document_id, doc_type, index):
         pass
 
 
+def serialize_public_enrolled_user(serialized_enrolled_user):
```

Comment:
Can you add a test for this, similar to https://github.com/mitodl/micromasters/blob/master/search/indexing_api_test.py#L404?